Source/WebCore/css/CSSProperties.json

```diff
@@ -635,6 +635,42 @@
                 "url": "https://drafts.csswg.org/css-fonts-3/#font-synthesis-prop"
             }
         },
+        "font-synthesis-weight": {
+            "inherited": true,
+            "codegen-properties": {
+                "custom": "All",
```

Comment:
I don't see why these need custom handlers, they seem like ordinary font properties that could be handled with "font-property": true codegen.

The font-synthesis shorthand has special mapping but it should be refactored so it is implemented in terms of these properties, not the other way round.